backend/organization/models/members.py

```diff
@@ -114,3 +114,68 @@ class Meta:
 
     def __str__(self):
         return "User %d member for organization %s" % (self.user.id, self.organization.name)
+
+class MembershipRequests(models.Model):
+    
+    id = models.AutoField(help_text='identifier of a join request'
+                                , primary_key=True
+                                ,verbose_name="join request Id")
+
+
+    user = models.ForeignKey(
+        User,
+        verbose_name="feedback_user",
+        help_text="Points to the user who sent the request",
+        on_delete=models.CASCADE,
+        null=False
+    )
+
+    target_membership_type = models.SmallIntegerField(choices=[(x.value,x.name) for x in MembershipTarget]
+                                                     ,null=False
+                                                     ,verbose_name="Membership Request Target Type",
+                                                      help_text="The type of entity a user requested to join (organization, project...)")
+
+    target_project =  models.ForeignKey(Project
+                                            ,null=True
+                                            ,on_delete=models.CASCADE
+                                            ,help_text="Points to the requested project")
+
```

Comment:
You are missing `verbose_name` here. Django doesn't require it but to be consistent with the rest of the code I would recommend adding them. 

Same for `related_name` on `ForeginKey` relationships. If you don't define related_name django automatically handles it but for consistency purposes I guess doesn't hurt to add a related_name too. You can look examples for Idea or Project tables to define custom `related_names`